src/beautifiers/prettier.coffee

```diff
@@ -0,0 +1,41 @@
+"use strict"
+
+Beautifier = require('./beautifier')
+prettier = require("prettier")
+
+module.exports = class Prettier extends Beautifier
+  name: "Prettier"
+  link: "https://github.com/prettier/prettier"
+  options: {
+    _:
+      tabWidth: "indent_size"
+      useTabs: "indent_with_tabs"
```

Comment:
I recommend something like https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/prettydiff.coffee#L10-L13

```coffeescript
useTabs: ["indent_with_tabs", "indent_char", (indent_with_tabs, indent_char) ->
  return (indent_with_tabs is true) or (indent_char is "\t")
]
```

which is the following in JavaScript:

```javascript
 useTabs: [
    "indent_with_tabs",
    "indent_char",
    function(indent_with_tabs,
    indent_char) {
      return (indent_with_tabs === true) || (indent_char === "\t");
    }
  ]
```